src/ts/forwarders/forwarder.ts

```diff
@@ -43,20 +43,26 @@ export interface DeploymentParams {
 }
 
 export interface RelayHubCallData {
-  to: string;
+  to?: string;
   data: string;
 }
 
-export interface ProxyAccountCallData extends RelayHubCallData {
+export interface ProxyAccountCallData {
+  to?: string;
   value?: BigNumberish;
+  data?: string;
```

Comment:
why is data optional here but not relayhub?

Reply:
When you use the proxy account you can send a value, but no data - just a normal value transfer. When you use the hub you can send data, but not value.